tpm2_pytss/types.py

```diff
@@ -4,8 +4,10 @@
 
 from ._libtpm2_pytss import ffi, lib
 
+from tpm2_pytss.utils import CLASS_INT_ATTRS_from_string, TPM2B_unpack
```

Comment:
Mixed import styles, relative on line 5, absolute on like 7

```suggestion
from .utils import CLASS_INT_ATTRS_from_string, TPM2B_unpack
```

Reply:
Darn IDE